.github/workflows/doc-index-updater-release.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
       - name: Fetch image for tagged commit
+        id: fetch-image
+        continue-on-error: true
         working-directory: ./products/medicines/doc-index-updater
         run: |
           TAG="$(git rev-parse --short=7 $GITHUB_SHA)"
```

Comment:
The release workflows need the Docker step added too

Reply:
done